src/validators.rs

```diff
@@ -41,6 +42,34 @@ pub struct StakeStats {
     pub count: u64,
 }
 
+/// Reward for validator
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Reward {
```

Comment:
maybe put this in it's own module so that we can reuse it for other routes?

Reply:
What would be a good name for a module for shared responses.  There doesn't seem to be an existing use of this pattern in this crate so far? Should we consider moving all the responses into a `models` module and leaving the endpoints within the module structure now?

Reply:
yeah

Reply:
Ok great will refactor this now. I will start with a single `models` module for now.

Reply:
yup doing them piecemeal is ok too doesn't all have to happen here

Reply:
All done, have a look.